web/projects/ui/src/app/routes/portal/routes/marketplace/components/controls.component.ts

```diff
@@ -78,7 +78,11 @@ import { ApiService } from 'src/app/services/api/embassy-api.service'
         appearance="secondary-grayscale"
         (click)="showService()"
       >
-        {{ 'View Installed' | i18n }}
+        {{
+          'View ' +
+            localPkg.stateInfo.state.charAt(0).toUpperCase() +
```

Comment:
Pretty sire there is a titlecase pipe for this. Also does it even need to be capitalized? I think we have been lower casing the second word

Reply:
👍 titlecase pipe
its quite varied and inconsistent across buttons actually, seems to depend on the phrase